beacon-chain/core/epoch/epoch_processing.go

```diff
@@ -21,6 +23,13 @@ import (
 	"github.com/prysmaticlabs/prysm/v3/proto/prysm/v1alpha1/attestation"
 )
 
+var (
+	activationQueueCount = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
this isn't the correct place for such a metric. This is a stateless method, so it can be called for any slot/epoch.

Reply:
good point, I forgot the replay case. Will clean them up